cas/store/src/Data/CAS/ContentStore.hs

```diff
@@ -716,13 +716,13 @@ removeAlias store alias =
         [":hash" SQL.:= hash]
 
 -- | List all aliases and the respective items.
-listAliases :: MonadIO m => ContentStore -> m [(Alias, Item)]
+listAliases :: MonadIO m => ContentStore -> m [(Alias, Item, ContentHash)]
 listAliases store =
   liftIO . withStoreLock store $
-    fmap (map (second Item)) $
+    fmap (map (\(a,b,c) -> (a, Item b, c))) $
       SQL.query_
         (storeDb store)
-        "SELECT name, dest FROM aliases"
+        "SELECT name, dest, hash FROM aliases"
```

Comment:
Where is the change to the column name coming from? Couldn't seem to find the related change.

Reply:
I think this was @mknorps's change. If I understand it, she's just adding an extra query field to be able to run the new test above.